app/index.html

```diff
@@ -62,9 +64,19 @@ <h3 class="text-muted">hangman</h3>
         <div class="letter-button" onclick="pickLetter(this)">Y</div>
         <div class="letter-button" onclick="pickLetter(this)">Z</div>
        </div>
-        <p><a class="btn btn-lg btn-success" href="#">Splendid! <span class="glyphicon glyphicon-ok"></span></a></p>
+        </div>
+        <div class="col-sm-4">
+          <div class="hangman-box">
+            <p>Spot for hangman!</p>
+          </div>
+        </div>
+        <div class="col-sm-4">
+          <div class="graveyard">
```

Comment:
I was thinking that as well. I will make those changes. Once I get home from work.